ivadomed/models.py

```diff
@@ -1195,7 +1223,7 @@ def __init__(self, in_channels, gating_channels, inter_channels=None, dimension=
             bn = nn.BatchNorm2d
             self.upsample_mode = 'bilinear'
         else:
-            raise NotImplementedError
+            raise NotImplemented
```

Comment:
what is the reason for changing that?

Reply:
these lines come from the source code of [GridAttentionBlock] (https://github.com/ozan-oktay/Attention-Gated-Networks/blob/eee4881fdc31920efd873773e0b744df8dacbfb6/models/layers/grid_attention_layer.py#L44) which I think they want to expand their attention gate block, or for assessing their code. 
i think that the previous person who implemented the 3D attention module left this line intentionally as a reminder to implement the 2D or maybe check regularly the attention Gate source code for its additional feature.

Reply:
https://help.semmle.com/wiki/display/PYTHON/NotImplemented+is+not+an+Exception

Reply:
> https://help.semmle.com/wiki/display/PYTHON/NotImplemented+is+not+an+Exception

👍